playbooks/openstack-create_network.yml

```diff
@@ -24,3 +24,16 @@
         - name: show network stack output
           debug:
             var: network_stack_create_output
+
+    - name: Get the VIP from the network stack
+      command: openstack stack output show {{ deploy_on_openstack_network_stackname }} vip -c output_value -f value
+      changed_when: false
+      register: _network_stack_vip_output
+
+    - name: Add vip with cidr in extravars
+      delegate_to: localhost
+      blockinfile:
```

Comment:
I think you need to use a different marker here than what is used in the ses-osh role. Other wise the two `blockinfile` calls with conflict.
See: https://docs.ansible.com/ansible/latest/modules/blockinfile_module.html
"When more then one block should be handled in one file you must change the marker per task"

Reply:
Fixed